packages/react/src/components/Dropdown/index.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { Dropdown } from '..';
+import { Item } from '..';
+
+const defaultProps = () => ({
+  id: `dropdown-${Math.random()}`,
+  onChange: action('onChange'),
+});
+
+storiesOf('Dropdowns/Dropdown', module)
```

Comment:
```suggestion
const stories = storiesOf('Dropdown', module)
```

Could you follow the same pattern as for every story?

- Use `const stories = `
- Add Heading
- Have a Playground story with as many knobs as possible as long as they make sense

Reply:
Done, yeah sorry didn't notice the patter.
I think it would be possible to implement a decorator to do it for us. Will add it to my TODO and take care of it when have time :)